src/toil/jobStores/azureJobStore.py

```diff
@@ -192,6 +195,33 @@ def deleteJobStore(self):
     def getEnv(self):
         return dict(AZURE_ACCOUNT_KEY=self.account_key)
 
+    @staticmethod
+    def readFromUrl(url, writable):
+        blobService, containerName, blobName = AzureJobStore._extractBlobInfoFromUrl(url)
+        blobService.get_blob_to_file(containerName, blobName, writable)
+
+    @staticmethod
+    def writeToUrl(readable, url):
+        blobService, containerName, blobName = AzureJobStore._extractBlobInfoFromUrl(url)
+        blobService.put_block_blob_from_file(containerName, blobName, readable)
+        blobService.get_blob(containerName, blobName)
+
+    @staticmethod
+    def _extractBlobInfoFromUrl(url):
+        """
+        :return: (blobService, containerName, blobName)
+        """
+        netloc = url.netloc.split('@')
+        containerName, accountName = netloc[0], netloc[1].split('.blob.core.windows.net')[0]
+        blobName = url.path.split('/')[1]
```

Comment:
Are you doing this because path starts with a '/'. If so that this is a convoluted way to get rid of the leading slash.